backend/routes/projectById.js

```diff
@@ -0,0 +1,34 @@
+const { parse } = require("url");
+
+const { missingProperty, invalidProperty } = require('../src/utils/errors');
+const stringify = require('../src/utils/stringify');
+const getAllProjects = require('../src/components/projects/getAllProjects');
+
+module.exports = async (req, res) => {
+  const { query: { project: projectParam } } = parse(req.url, true);
+
+  if (!projectParam) {
```

Comment:
Truthy comparisons are for N00bs, be more explicit. I guess you just want to check for 'undefined'

Reply:
I wouldn't agree with it. I know that the return of this will be a string and I want to go into this if if this string is empty. I'd say being explicit in here isn't really necessary

Reply:
Ofc it isn't necessary, but it makes things easier to understand for developers other that the one writing it.